lib/workers/repository/process/lookup/index.ts

```diff
@@ -752,7 +754,8 @@ export async function lookupUpdates(
         rollbackPrs: config.rollbackPrs,
         isVulnerabilityAlert: config.isVulnerabilityAlert,
         updatePinnedDependencies: config.updatePinnedDependencies,
-        unconstrainedValue,
+        unconstrainedValue:
+          !!config.lockedVersion && is.undefined(config.currentValue),
```

Comment:
is the const out of scope now?

Reply:
Yes

Reply:
What's the benefit of this change? We're now left with non-DRY code. Will that be fixed later?

Reply:
Just removed it completely for simplicity. If anyone ever care about this logged field, this could be inferred from two others that still exist.